aws/vpc/Makefile

```
@@ -0,0 +1,79 @@
+
+# Default values
+ENV ?= production
+
+# Colors for output
+RED := \033[0;31m
+GREEN := \033[0;32m
+YELLOW := \033[0;33m
+BLUE := \033[0;34m
+NC := \033[0m # No Color
+
+.PHONY: help init plan apply destroy validate fmt check clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)